packages/valory/skills/decision_maker_abci/behaviours/sampling.py

```diff
@@ -111,8 +119,23 @@ def _sampled_bet_idx(self, bets: List[Bet]) -> int:
 
     def _sample(self) -> Optional[int]:
         """Sample a bet, mark it as processed, and return its index."""
-        available_bets = list(filter(self.processable_bet, self.bets))
+        # modify time "NOW" in benchmarking mode
+        if self.benchmarking_mode.enabled:
+            safe_voting_range = (
+                self.params.opening_margin + self.params.safe_voting_range
+            )
+            self.synced_timestamp = self.shared_state.get_simulated_now_timestamp(
+                self.bets, safe_voting_range
+            )
```

Comment:
Where? Neither the `synced_timestamp` property nor how the `now` is set has been edited.

Reply:
in the benchmarking mode we might not want the same logic everytime this property is accessed because in each Round we might introduced a different behaviour, thus the current implementation is the most simple and compatible solution with the new time behaviour in Benchmarking mode.